sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1252,6 +1252,10 @@ def task_worker(self):
         for namespace in self.namespaces:
             self.wait_for_port_config_done(namespace)
 
+        logical_port_list = self.port_mapping.logical_port_list
+        for lport in logical_port_list:
+            self.update_port_transceiver_status_table_sw_cmis_state(lport, CMIS_STATE_UNKNOWN)
```

Comment:
@mihirpat1 do we need to take care of Xcvrd restart?

Reply:
We don't need to explicitly handle xcvrd restart case since the below code will set the `cmis_state` to `CMIS_STATE_READY` and prevent link flap if the module is already initialized.
https://github.com/sonic-net/sonic-platform-daemons/blob/d513eca54f27c15fa384c927a5fd0f01be1353e3/sonic-xcvrd/xcvrd/xcvrd.py#L1390

Also, the current nightly behavior is to clear `cmis_state` after xcvrd restart. Hence, this should not cause additional change in behavior.